src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -196,6 +196,10 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
         val augmentedClusterRequest = augmentClusterRequest(serviceAccountInfo, googleProject, clusterName, userEmail, clusterRequest)
         val clusterImages = processClusterImages(clusterRequest)
         val clusterFuture = for {
+          // Metrics
+          _ <- Metrics.newRelic.incrementCounterFuture("clusterRequests")
```

Comment:
Shall we rename this to something like `numberOfCreateClusterRequest`. It's not obvious what `clusterRequests` means by itself.

Same comment for the other metrics, something like ``numberOfCreateClusterRequestWithWelder`?